src/pages/options/options.js

```diff
@@ -1,22 +1,12 @@
 import ReactDOM from 'react-dom'
 import React, { useState } from 'react'
 import { css, cx } from 'linaria'
-import {
-  PocketLogo,
-  PocketNewtab,
-  AndroidLogo,
-  IphoneIpad,
-  Mac
-} from 'elements/icons'
```

Comment:
I removed these icons because they were causing render issues. Will address as part of the icons audit or when we are updating the options page with the new design.